main.rb

```diff
@@ -16,8 +16,22 @@ def initialize(name, email, squad, nickname)
     @nickname = nickname
   end
 
-  def summit(will_work, reason)
-    will_work == true ? "Go to summit" : "#{name} não vai por esse motivo: #{reason}"
+  def summit(will_work, reason="")
+    jobs = ['Chopeira', 'Credenciamento', 'Gerenciamento de Filas', 'Hosts dos Palestrantes']
+
+    if will_work == true
+      rdoerJob = job_select(jobs)
+      puts "#{name} vai no Summit e vai trabalhar com #{rdoerJob}"
+    else 
+      "#{name} não vai no Summit por esse motivo: #{reason}"
+    end
+  end
+
+  private def job_select(jobs)
```

Comment:
Aqui você poderia declarar o `private` uma linha acima, indicando que todos os métodos dali para baixo serão privados:

```suggestion
  private 
  
  def job_select(jobs)
```

Reply:
Make sense.